### PR TITLE
Description textarea supports HTML

### PR DESCRIPTION
**Pull request by martinda**
Added support of the HTML-style description (coded copied from the Jenkins built-in StringParameterDefinition/config.jelly). Visually inspected using Jenkins 1.597.

### CODE REVIEW

File: src/main/resources/hudson/plugins/validating_string_parameter/ValidatingStringParameterDefinition/config.jelly [hunk 1]
```diff
@@ -22,22 +22,23 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 -->
 
+<?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define"
 	xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form"
 	xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
     <f:entry title="${%Name}" help="/help/parameter/name.html">
-		<f:textbox field="name" />
+		<f:textbox name="parameter.name" value="${instance.name}" />
```
Comment (Vlatombe) [inline src/main/resources/hudson/plugins/validating_string_parameter/ValidatingStringParameterDefinition/config.jelly]: why did you change this?

**Comment by Vlatombe** [comment]
Overall I like the change, but there are some unrelated changes (field -> name) that should be removed from the pr

File: src/main/resources/hudson/plugins/validating_string_parameter/ValidatingStringParameterDefinition/config.jelly (hunk 1 above)
Comment (martinda) [inline src/main/resources/hudson/plugins/validating_string_parameter/ValidatingStringParameterDefinition/config.jelly]: I was trying to keep the whole file aligned with the real modification on line 42, which uses "name" and "value" rather than "field". Other than that, no particular reason. I have copied the feature from line 36 in: https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/hudson/model/StringParameterDefinition/config.jelly#L36

Comment (Vlatombe) [inline src/main/resources/hudson/plugins/validating_string_parameter/ValidatingStringParameterDefinition/config.jelly]: It is usually preferred to use data-binding (field=...) rather than name and value.

**Comment by martinda** [comment]
I made those changes. Let me know if there is something else I need to do for this PR to be accepted.

**Comment by Vlatombe** [comment]
Merged as cec83057d3feb8246b333eff090ae80d657cd9aa
